operations/mimir-mixin/dashboards/ruler.libsonnet

```
@@ -49,12 +51,19 @@ local filename = 'mimir-ruler.json';
           else
             'Writes to ingesters - RPS'
         ) +
-        $.statPanel(
-          local query =
+        $.newStatPanel(
+          local selectors = $.jobSelector($._config.job_names.ruler) + [utils.selector.eq('operation', '/cortex.Ingester/Push')];
+          local classicQueries = $.ncSumHistogramCountRate('cortex_ingester_client_request_duration_seconds', selectors);
```

nit, naming is a bit misleading
```suggestion
          local baseQuery = $.ncSumHistogramCountRate('cortex_ingester_client_request_duration_seconds', selectors);
```